### PR TITLE
Adding note highlighting endpoints not supported by ModelMesh

### PR DESCRIPTION
**Pull request by grainnejenningsRH**
Adds a note to modules/ref-inference-endpoints.adoc, highlighting the REST endpoints that ModelMesh doesn't support.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/ref-inference-endpoints.adoc [hunk 1]
```diff
@@ -216,6 +216,15 @@ endif::[]
 * `v2`
 --
 
+[NOTE]
+====
+ModelMesh does not support the following REST endpoints:
+
+* `v2/health/live`
+* `v2/health/ready`
+* `v2/models/<model_name>[/versions/]/ready`
```
Comment (chtyler) [inline modules/ref-inference-endpoints.adoc:225]: * `v2/models/<model_name>[/versions/]/ready`

Does the plural versions imply that multiple version numbers can appear in the endpoint consecutively, or does it imply that only one version of multiple versions can appear?

**Review by chtyler — CHANGES_REQUESTED** [review]
One minor clarification.

File: modules/ref-inference-endpoints.adoc (hunk 1 above)
Comment (chtyler) [reply modules/ref-inference-endpoints.adoc:225]: It might not actually be a placeholder. The square brackets confused me a bit.

Comment (grainnejenningsRH) [reply modules/ref-inference-endpoints.adoc:225]: Per separate discussion - we'll add this as-is for the moment, and follow up in the new year on whether the formatting of `/versions/` needs any improvement.

**Pull request merged**